resources/js/bots/transactor_personal/bot.js

```diff
@@ -62,13 +62,25 @@ function amountParameterBox(params, context) {
 
     var sliderValue = params["bot-db"]["sliderValue"] || 0;
 
-    status.setDefaultDb({
-        transaction: txData,
-        calculatedFee: calculateFee(sliderValue, txData),
-        feeExplanation: getFeeExplanation(sliderValue),
-        sliderValue: sliderValue
-    });
+    try {
+
+        status.setDefaultDb({
+            transaction: txData,
+            calculatedFee: calculateFee(sliderValue, txData),
+            feeExplanation: getFeeExplanation(sliderValue),
+            sliderValue: sliderValue
+        });
 
+    } catch (err) {
```

Comment:
How about only having `calculateFee` in try/catch so that the call to `setDefaultDb` is not duplicated?

Reply:
I didn't put much thought to that as I was simply making sure that `transactor_personal`/`transactor_group` code is the same and behaving consistently, I can certainly do that.